test/e2e/specs/specs-playwright/wp-plans_purchases-spec.ts

```diff
@@ -0,0 +1,119 @@
+import {
+	DataHelper,
+	LoginFlow,
+	setupHooks,
+	SidebarComponent,
+	PlansPage,
+	IndividualPurchasePage,
+	CartCheckoutPage,
+	BrowserHelper,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+/**
+ * Constants
+ */
+
+const userOnPremiumPlan = 'defaultUser';
+
+describe( DataHelper.createSuiteTitle( 'Plans: Purchases' ), function () {
+	let page: Page;
+	let plansPage: PlansPage;
+	let purchasesPage: IndividualPurchasePage;
+	let cartCheckoutPage: CartCheckoutPage;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	describe( 'Initial navigation', function () {
+		it( 'Log In', async function () {
+			const loginFlow = new LoginFlow( page, userOnPremiumPlan );
+			await loginFlow.logIn();
+		} );
+
+		it( 'Navigate to Plans from sidebar', async function () {
+			const sidebarComponent = new SidebarComponent( page );
+			await sidebarComponent.gotoMenu( { item: 'Upgrades', subitem: 'Plans' } );
+		} );
+
+		it( 'Click on the "My Plan" tab', async function () {
+			plansPage = new PlansPage( page );
+			await plansPage.clickNavigationTab( 'My Plan' );
+		} );
+
+		it( 'The Premium plan is listed as current plan on "My Plan" tab', async function () {
+			await plansPage.validatePlanInMyPlan( 'Premium' );
+		} );
+
+		it( 'Click on the "Plans" navigation tab', async function () {
+			await plansPage.clickNavigationTab( 'Plans' );
+		} );
+	} );
+
+	// The manage button is currently broken on mobile - see issue #54858. TODO: remove the if check when that issue is fixed.
+	if ( BrowserHelper.getViewportName() === 'desktop' ) {
```

Comment:
Not a nit or a suggestion to change, just sharing a neat approach I found the other day looking for 'conditional skip' in Jest: [link](https://stackoverflow.com/a/60438234).

Reply:
Oh I love this!! I will do something, maybe like "describeDesktopOnly"